doc/Analytics.xml

```diff
@@ -4368,6 +4368,84 @@ xmlns:acme="http://www.acme.com/schema"&gt;
     </tt:ParentTopic>
   </tt:Messages>
 </tt:RuleDescription>
+]]></programlisting>
+    </section>
+    <section>
+      <title>Object Track Aggregation</title>
+      <para>The Object Track Aggregation rule generates an object track aggregation event for below
+        listed scenarios </para>
+      <itemizedlist>
+        <listitem>
+          <para> Optionally, Initial aggregation after an object appears OR an Intermediate aggregation in the field of view 
+            or an Intermediate aggregation while the object is present in the field of view.</para>
+        </listitem>
+        <listitem>
+          <para> Final aggregation when the object is no longer visible in the field of view. </para>
+        </listitem>
+      </itemizedlist>
+      <variablelist role="op">
+        <varlistentry>
+          <term>Parameters</term>
+          <listitem>
+            <para role="param">ClassFilter [tt:StringList]</para>
+            <para role="text">List of classes to be aggregated.</para>
+            <para role="param">ConfidenceLevel - optional [xs:float]</para>
+            <para role="text">Minimum confidence level of object classification for generating
+              aggregation event. </para>
+          </listitem>
+        </varlistentry>
```

Comment:
Suggest to add

    ReportTimeInterval - optional [xs:duration]
    Optional time interval to control update frequency. 
    If omitted only final information will be provided.

This allows a client to define after which time it wants to have an update. If not defined only a final information will be provided when the object has vanished which may take hours in case of loitering.

Reply:
We agree with this comment - but we would like the vendor to have the choice to send 'intermediates' or just the 'final' update if the parameter is NOT configured or omitted.

As we understand **ReportTimeInterval** should be interpreted as the maximum time interval before which an update need to be sent?

Reply:
Make the parameter optional and let the device signal via options whether it supports the parameter.

Reply:
Similar "ReportTimeInterval" parameter used in line crossing and occupancy counting rules (Sections 5 and 6) without the corresponding options, I am a bit unclear why its expected/required here and can't have it as just the parameter without the option?

Reply:
ReportTimeInterval made optional